### PR TITLE
[OBSDEF-4676] - Add direct access support in support assist

### PR DESCRIPTION
**Pull request by fyang86**
## Purpose
[OBSDEF-4676](https://jira.cec.lab.emc.com/browse/OBSDEF-4676)
add direct access support in support assist controller in decks, decks change in https://eos2git.cec.lab.emc.com/atlantic/decks/pull/101

## PR checklist
- [x] make test passed
- [x] make build passed
- [x] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

### CODE REVIEW

File: supportassist/templates/supportassist-cr.yaml [hunk 1]
```diff
@@ -26,6 +26,10 @@ spec:
   gateways:
   {{toYaml .Values.gateways | indent 2 | trim}}
   {{- end }}
+  {{- if .Values.directEndpoints }}
+  directEndpoints:
+  {{toYaml .Values.directEndpoints | indent 2 | trim}}
+  {{- end }}
```
Comment (eddiepavkovic) [inline supportassist/templates/supportassist-cr.yaml:32]: we shouldn't require this from the customer.  this is an internal setting inside ese.

File: supportassist/values.yaml [hunk 2]
```diff
@@ -51,6 +51,10 @@ useGateways: true
 #    port: 9443
 #    priority: 20
 
+# directEndpoints: - required field to pass using --set
+#  - hostname: esrs3-corestg.isus.emc.com
+#    priority: 20
+
```
Comment (eddiepavkovic) [inline supportassist/values.yaml:57]: we shouldn't require this from the customer.  they don't know what to set here.  its built internally into the ESE code

**Review by eddiepavkovic — CHANGES_REQUESTED** [review]

**Comment by eddiepavkovic** [comment]
converting to draft,  we can't ask customers to provide the directEndpoints params,  that's internal to dellemc

File: supportassist/values.yaml (hunk 2 above)
Comment (fyang86) [reply supportassist/values.yaml:57]: so we shouldn't read the direct endpoint from support assist CR, just use it inside the ese code right?

**Comment by eddiepavkovic** [comment]
closing this,  confirmed with Vaishali today that we only need to set `useGateways = false` and it'll auto use the direct connect setting.  no need to set endpoints its already internally configured.

**Pull request closed without merging**